setup.py

```diff
@@ -39,7 +39,7 @@
     ext_modules=[
         Extension("intbitset",
                   ["src/intbitset.c", "src/intbitset_impl.c"],
-                  extra_compile_args=['-O3', '-march=native']
```

Comment:
We could use `-march=<specific architecture` instead and set it to something which is up-to-date enough to have SSEx (or something else) but is not some crazy new system.